athena-neptune/docs/aws-glue-sample-scripts/manual/sample-cli-rdf.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# id is the reserved column for vertex tables
+# id, from, to are the reserved columns for edge tables
+
+echo $1;
+echo $2;
+echo $3;
```

Comment:
remove the 3rd parameter, its not needed and update the script accordingly

Reply:
removed